### PR TITLE
Prevent `simply_harvest.sc` from using up blocks in either hand

### LINKED ISSUE #351: simply_harvest off-hand block deletetion
**Describe the bug**
Off hand blocks are deleted while harvesting with simply_harvest

**To Reproduce**
1 place blocks in your off hand
2 harvest fully grown crops

**Expected behavior**
blocks in the off hand are not affected, moved to inventory or dropped if necessary. 

**Screenshots**
N/A

**Environment**
 - Carpet version: fabric-carpet-1.19.2-1.4.84+v221018
 - Minecraft version: 1.19.2

### PR DESCRIPTION
**Pull request by rv3r**
Fix #351

### CODE REVIEW

**Review by altrisi — APPROVED** [review]
Tested and fixes the issue! Don't know if it may be fragile in cases where multiple things happen per tick, but I think it's fine as that's unlikely anyway.

File: programs/survival/simply_harvest.sc [hunk 1]
```diff
@@ -16,6 +16,9 @@ global_seeds = {
     'nether_wart' -> 'nether_wart'
 };
 
+// variable indicating that harvesting occurred
+global_harvesting = false;
+
```
Comment (Ghoulboy78) [inline programs/survival/simply_harvest.sc:21]: Where in the code do we actually query this variable tho?

Comment (altrisi) [reply programs/survival/simply_harvest.sc:21]: In __on_player_placing_block (ran after the main event handler if offhand) and in the 0-scheduled task (where it's just set back to false).

Comment (Ghoulboy78) [reply programs/survival/simply_harvest.sc:21]: Mah bad, didn't read properly

**Pull request merged**